Settings specific for MariaDB specific are defined in appropriate section in
/etc/my.cnf.d/server.cnf (part of mariadb-server).
It doesn't matter that we set these settings only for [mysqld] here,
because they will be read and used in mysqld_safe as well.
Settings in my.cnf are generic for all MySQL implementations because of
conflict issues.
RHBZ#1003115

diff -up mariadb-5.5.32/support-files/rpm/server.cnf.mariaconf mariadb-5.5.32/support-files/rpm/server.cnf
--- mariadb-5.5.32/support-files/rpm/server.cnf.mariaconf	2013-09-02 14:17:37.277833263 +0200
+++ mariadb-5.5.32/support-files/rpm/server.cnf	2013-09-02 14:18:00.638810223 +0200
@@ -23,6 +23,8 @@
 # If you use the same .cnf file for MySQL and MariaDB,
 # you can put MariaDB-only options here
 [mariadb]
+log-error=/var/log/mariadb/mariadb.log
+pid-file=/var/run/mariadb/mariadb.pid
 
 [mariadb-5.5]
 
